### PR TITLE
Updates for 63-3 and 63A Sections

### PR DESCRIPTION
**Pull request by bsahin**

### CODE REVIEW

File: sp800-63a/1_Evidence.md [hunk 1]
```diff
@@ -4,9 +4,26 @@ title:  Identity Evidence
...
... combinations of core and supplemental attributes provide varying levels of effectiveness, sensitivity, availability and reliability. An assessment of such metrics for core and supplemental identity attributes, e.g. name, address, date and place of birth, mother’s maiden name and gender, can be found in NASPO-IDPV-60 by the American Security Products Organization’s Identity Proofing and Verification (IDPV) working group (NASPO-IDPV-60. Establishment of Core Identity Attribute Sets and Supplemental Identity Attributes, Report of the IDPV Identity Resolution Project, Feb. 17, 2014). The analysis conducted in this study provides interesting results in terms of the effectiveness of combining attributes for resolution and also shows that there are diminishing returns to increasing the number of core attributes collected from applicants, as evidenced in the following table. Organizations that seek a balance between the effectiveness of resolution and an increasing list of collected attributes can review such studies for information. Its inclusion here is not intended to endorse the IDPV standard, but instead illustrate how attribute sets may be balanced or combined to support resolution.
```
Comment (jimfenton) [inline sp800-63a/1_Evidence.md:11]: I can't find a source for NASPO-IDPV-60 online anywhere, and it sounds like agencies will need it to evaluate resolution. Please provide a full bibliographic reference to it.

File: sp800-63a/1_Evidence.md [hunk 2]
```diff
@@ -4,9 +4,26 @@ title:  Identity Evidence
 collection: 63A
 permalink: /63A/Evidence
 ---
-Identity evidence comprises physical or digital artifacts used by a subscriber to prove the real-world existence of a claimed identity. Evidence required by the agency and provided by the applicant must contain features that support both validation (confirmed to be genuine and accurate) and verification (demonstrated to associated with the subscriber) at the appropriate strengths for a defined use case.
+Identity evidence comprises physical or digital artifacts used by a subscriber to prove the real-world existence of a claimed identity. Evidence required by the agency and provided by the applicant must contain features that enable the agency to resolve the identity and support both validation (confirmed to be genuine and accurate) and verification (demonstrated to associated with the subscriber) at the appropriate strengths for a defined use case.
```
Comment (jimfenton) [inline sp800-63a/1_Evidence.md:7]: Does "strengths" mean IALs? If so, we should be consistent in terminology.

File: sp800-63a/1_Evidence.md [hunk 3]
```diff
@@ -4,9 +4,26 @@ title:  Identity Evidence
...
+| Middle Initial |  |  |  |  | Y |
+| Full DOB |  |  | Y |  | Y |
+| Partial DOB (YYYY or MMDD) |  | Y |  |  |  |
+| Year of Birth | Y |  |  |  |  |
+| Partial Address (Zip or City/State) |  | Y |  | Y |  |
+| Full Address |  |  |  |  | Y |
+| Last 4 Digits of SSN |  |  |  | Y | Y |
+| Estimated Resolution Effectiveness (%) | 2.56 | 98.73 | 96.29 | 96.65 | 99.89 |
 
 The following table provides examples of evidence from each category of strength defined in the aforementioned identity evidence quality table in 800-63A.
```
Comment (jimfenton) [inline sp800-63a/1_Evidence.md:28]: Don't understand what is meant by "category of strength". I thought this referred to resolution effectiveness, not overall strength (IAL).

File: sp800-63a/2_Validation.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ Identity validation is made up of two process steps: confirming the evidence is
...
 
-Agencies should also examine the evidence visually. Techniques for remote validation should be comparable to in-person validation. Table 5-2 in SP800-63A describe methods for validating identity evidence at each strength.
+Agencies should also examine the evidence visually. Techniques for remote validation should be comparable to in-person validation. Table 5-2 in SP800-63A describe methods for validating identity evidence. For remote validation, using offline data records such as those in the above example is an option. Cryptographic capabilities may also be helpful in performing identity validation remotely. It should be acknowledged that for IAL2 and IAL3, the ability to validate evidence remotely is limited.
```
Comment (jimfenton) [inline sp800-63a/2_Validation.md:13]: Suggest removing "for IAL2 and IAL3". This is true in general, and there is no requirement to validate evidence at IAL1.

File: sp800-63a/introduction.md [hunk 5]
```diff
@@ -16,3 +16,4 @@ Identity proofing is the process by which a Credential Service Provider (CSP) co
 
 ![Identity Proofing Process Flow]({{site.baseurl}}/img/ProofingProcess.png)
 
+Please note that while the diagram above illustrates the identity proofing process as a roadmap composed of concrete successive steps, it is meant more generally to convey the relationships between the different objectives and their definitions. In reality, collection of evidence for these steps can often take place concurrently, with no additional evidence collected beyond the initial steps in enrollment, which is both used for identity resolution and validated.
```
Comment (jimfenton) [inline sp800-63a/introduction.md:19]: "and validated" -> "and validation"
Suggest also "In reality" -> "In practice"

File: sp800-63-3/2_acceptance.md [hunk 6]
```diff
@@ -18,7 +18,3 @@ In accordance with SP 800-63-3, a Digital Identity Acceptance Statement shall in
 
 Additional information included in the statement is up to individual agency discretion. 
 
-A Digital Identity Risk Acceptance Statement template is provided for agency use. The use of this template is not required, but is intended to provide agencies with a better idea of what the statement could look like. Agencies may wish to include additional information that is not present in this template, or less information, but the required information outlined above must be included in the final statement. 
-
-<a href="/SP800-63A-Resource-Center/img/Risk_Acceptance_Statement_Template_DRAFT.docx"> <button>Download the Digital Identity Risk Acceptance Statement Template</button> </a>
```
Comment (jimfenton) [inline sp800-63-3/2_acceptance.md:23]: The pull request should also remove the template itself as well.

**Review by jimfenton — CHANGES_REQUESTED** [review]
Several comments and questions included. Let me know if you want to discuss.

File: sp800-63a/1_Evidence.md (hunk 1 above)
Comment (bsahin) [reply sp800-63a/1_Evidence.md:11]: The IDPV Identity Resolution WG Report quoted above does not seem to be available online.

**Pull request closed without merging**